editor/import/scene_import_settings.cpp

```diff
@@ -594,6 +594,9 @@ void SceneImportSettings::open_settings(const String &p_path, bool p_for_animati
 	// Visibility
 	data_mode->set_tab_hidden(1, p_for_animation);
 	data_mode->set_tab_hidden(2, p_for_animation);
+	if (p_for_animation) {
+		data_mode->set_current_tab(0);
+	}
```

Comment:
I wonder if this should be conditional or should always be done.

WDYT @godotengine/import?